lerobot/common/robot_devices/cameras/intelrealsense.py

```diff
@@ -69,12 +71,13 @@ def save_images_from_cameras(
     Initializes all the cameras and saves images to the directory. Useful to visually identify the camera
     associated to a given camera index.
     """
-    if camera_ids is None:
-        camera_ids = find_camera_indices()
+    if len(camera_ids) == 0:
+        camera_ids = find_cameras_info()
 
     print("Connecting cameras")
     cameras = []
     for cam_idx in camera_ids:
```

Comment:
I agree, this was because it was copied from a very similar function for opencv cameras (see [this discussion](https://github.com/huggingface/lerobot/pull/409#discussion_r1754228025)).
I'll wait for that discussion to resolve before changing it.